src/commands/platforms/create.ts

```diff
@@ -24,24 +24,79 @@ export default class PlatformCreate extends Command {
   static flags = {
     ...Command.flags,
     ...AccountUtils.flags,
-    auto_approve: flags.boolean(),
+    auto_approve: flags.boolean({
+      description: `${Command.DEPRECATED} Please use --auto-approve.`,
+      hidden: true,
+    }),
+    ['auto-approve']: flags.boolean(),
     type: flags.string({ char: 't', options: ['KUBERNETES', 'kubernetes', 'ECS', 'ecs'] }),
     host: flags.string({ char: 'h' }),
-    kubeconfig: flags.string({ char: 'k', default: '~/.kube/config', exclusive: ['service_token', 'cluster_ca_cert', 'host'] }),
-    aws_key: flags.string({ exclusive: ['awsconfig', 'kubeconfig', 'service_token', 'cluster_ca_cert', 'host'] }),
-    aws_secret: flags.string({ exclusive: ['awsconfig', 'kubeconfig', 'service_token', 'cluster_ca_cert', 'host'] }),
-    aws_region: flags.string({ exclusive: ['awsconfig', 'kubeconfig', 'service_token', 'cluster_ca_cert', 'host'] }),
-    service_token: flags.string({ description: 'Service token', env: 'ARCHITECT_SERVICE_TOKEN' }),
-    cluster_ca_cert: flags.string({ description: 'File path of cluster_ca_cert', env: 'ARCHITECT_CLUSTER_CA_CERT' }),
+    kubeconfig: flags.string({ char: 'k', default: '~/.kube/config', exclusive: ['service-token', 'cluster-ca-cert', 'host'] }),
+    aws_key: flags.string({
+      exclusive: ['awsconfig', 'kubeconfig', 'service-token', 'service_token', 'cluster-ca-cert', 'cluster_ca_cert', 'host'],
+      description: `${Command.DEPRECATED} Please use --aws-key.`,
+      hidden: true,
+    }),
+    ['aws-key']: flags.string({
+      exclusive: ['awsconfig', 'kubeconfig', 'service-token', 'service_token', 'cluster-ca-cert', 'cluster_ca_cert', 'host'],
+    }),
+    aws_secret: flags.string({
+      exclusive: ['awsconfig', 'kubeconfig', 'service-token', 'service_token', 'cluster-ca-cert', 'cluster_ca_cert', 'host'],
+      description: `${Command.DEPRECATED} Please use --aws-secret.`,
+      hidden: true,
+    }),
+    ['aws-secret']: flags.string({
+      exclusive: ['awsconfig', 'kubeconfig', 'service-token', 'cluster-ca-cert', 'host']
+    }),
+    aws_region: flags.string({
+      exclusive: ['awsconfig', 'kubeconfig', 'service-token', 'cluster-ca-cert', 'host'],
+      description: `${Command.DEPRECATED} Please use --aws-region.`,
+      hidden: true,
+    }),
+    ['aws-region']: flags.string({
+      exclusive: ['awsconfig', 'kubeconfig', 'service-token', 'cluster-ca-cert', 'host']
+    }),
+    service_token: flags.string({
```

Comment:
Remove unused flags service_token/cluster_ca_cert and their new dashed versions

Reply:
If it's okay with you, I'd really like to remove these in a separate commit, that way the resulting merge commits are relatively atomic, even though it kinda sucks to maintain the dead weight of these couple extra flags through this PR 😬

Reply:
Fine by me as long as we don't forget

Reply:
Gotcha- I recognize I still haven't made a ticket for taking care of that. Meant to make it after wrapping up this PR, but this has taken a fair bit longer than expected 😅